doc/sphinx-guides/source/developers/sql-upgrade-scripts.rst

```diff
@@ -2,40 +2,38 @@
 SQL Upgrade Scripts
 ===================
 
-The database schema for Dataverse is constantly evolving. As other developers make changes to the database schema you will need to keep up with these changes to have your development environment in working order. Additionally, as you make changes to the database schema, you must write SQL upgrade scripts when needed and communicate with your fellow developers about applying those scripts.
+The database schema for Dataverse is constantly evolving. As other developers make changes to the database schema you will need to keep up with these changes to have your development environment in working order. Additionally, as you make changes to the database schema, you must write SQL when needed and communicate with your fellow developers about making those SQLs.
 
 .. contents:: |toctitle|
 	:local:
 
-Location of SQL Upgrade Scripts
--------------------------------
+Location of SQL Upgrade Scripts and Flyway SQLs
+-----------------------------------------------
 
-``scripts/database/upgrades`` is the directory where we keep or SQL upgrade scripts.
+``scripts/database/upgrades`` is the directory where we keep SQL upgrade scripts - legacy way of upgrading schema.
```

Comment:
Couldn't those be migrated into Flyway someway, too? It would be really cool to get people currently on older releases onto the latest release as easy as possible.

Reply:
I think we should stick with the old way till the user is in the newest version, trying to be back-compatible IMO introduces too much mess, you would have to came up with some way to pass the previous app version and then make migration according to that. And the script which we have already does that.